components/brave_vpn/browser/connection/win/brave_vpn_wireguard_service/BUILD.gn

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import("//brave/build/config.gni")
+import("//build/toolchain/win/midl.gni")
+import("//build/util/process_version.gni")
+import("//chrome/process_version_rc_template.gni")
+
+assert(is_win)
+
+midl("brave_wireguard_manager_idl") {
+  sources = [ "brave_wireguard_manager_idl.idl" ]
+
+  writes_tlb = true
+}
+
+# This service needs to work with Windows 7, so explicitly setting the defines
```

Comment:
I don't doubt that this is correct, but I wonder why. Didn't we stop supporting Windows 7 at the beginning of the year?

Reply:
Im not sure, just did same as elevation service

Reply:
https://support.brave.com/hc/en-us/articles/11197967945613-Sunsetting-support-for-Windows-7-and-8-8-1

I do see the same code in `elevation_service` upstream. But I suspect it's outdated and will be removed soon.

I would try to remove the `config("winver")` block and see if that removal causes problems.

Reply:
done